microsoft-edge/webview2/reference/javascript/webview.yml

```diff
@@ -0,0 +1,199 @@
+        AddEventListenerOptions): void;
+      parameters:
+        - id: type
+          description: ''
+          type: string
+        - id: listener
+          description: ''
+          type: EventListenerOrEventListenerObject
+        - id: options
+          description: ''
+          type: boolean | AddEventListenerOptions
+      return:
+        type: void
+        description: ''
+  - name: postMessage(message)
+    uid: 'WebView2Script!WebView#postMessage:member(1)'
+    package: WebView2Script!
+    fullName: postMessage(message)
+    summary: >-
+      When the page calls `postMessage`<!-- -->, the `message` parameter is converted to JSON and is posted
+      asynchronously to the WebView2 host process. This will result in either the `CoreWebView2.WebMessageReceived`
+      event or the `CoreWebView2Frame.WebMessageReceived` event being raised depending on if `postMessage` is called
+      from the top level document in the WebView2 or a child frame.
+    remarks: |-
+        #### Examples
+
+        Post a message to the CoreWebView2:
+
+        ```TypeScript
+        const inTopLevelFrame = (window === window.parent);
+        if (inTopLevelFrame) {
+            // The message can be any JSON serializable object.
+            window.chrome.webview.postMessage({
+                myMessage: 'Hello from the script!',
+                otherValue: 1}
+            );
+            // A simple string is an example of a JSON serializable object
+            window.chrome.webview.postMessage("example");
+        }
+        ```
+    isPreview: false
+    isDeprecated: false
+    syntax:
+      content: 'postMessage(message: any) : void;'
+      parameters:
+        - id: message
+          description: The message to send to the WebView2 host. This can be any object that can be serialized to JSON.
+          type: any
+      return:
+        type: void
+        description: ''
+  - name: releaseBuffer(buffer)
+    uid: 'WebView2Script!WebView#releaseBuffer:member(1)'
+    package: WebView2Script!
+    fullName: releaseBuffer(buffer)
+    summary: >-
+      Call with the ArrayBuffer from the chrome.webview.sharedbufferreceived event to release the underlying shared
+      memory resource.
+    remarks: ''
+    isPreview: false
+    isDeprecated: false
+    syntax:
+      content: 'releaseBuffer(buffer: ArrayBuffer): void;'
+      parameters:
+        - id: buffer
+          description: An ArrayBuffer from the chrome.webview.sharedbufferreceived event.
+          type: ArrayBuffer
+      return:
+        type: void
+        description: ''
+  - name: 'removeEventListener(type, listener, options)'
+    uid: 'WebView2Script!WebView#removeEventListener:member(1)'
+    package: WebView2Script!
+    fullName: 'removeEventListener(type, listener, options)'
+    summary: >-
+      The standard EventTarget.removeEventListener method. Use it to unsubscribe to the `message` or

```

Comment:
```suggestion
      The standard `EventTarget.removeEventListener` method. Use it to unsubscribe to the `message` or
```

Reply:
They should be supported as the existing ones are rendered correctly.